### PR TITLE
Added necessary files for contributors

### PR DESCRIPTION
**Pull request by dhruvapte26**
Reference to Issue #45 
Created a:
- Code of conduct
- Pull request template
- Issue template
- contributing

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+## How to contribute?
+
+Do you wish to
+
+1. Add a series?
+2. Enhance the code?
+3. Solve bugs?
+
+We heartily welcome you! :tada:
```
Comment (athityakumar) [inline CONTRIBUTING.md]: A ❤️  would be more apt here than 🎉 here , I guess.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+You can see the issues [here](https://github.com/athityakumar/tvseries/issues). The workflow is given in the [README.md](https://github.com/athityakumar/tvseries/blob/master/README.md). 
+Now making a pull request is very easy! :pizza:
+
+1. Fork or clone this repository
+2. Create a new branch. You can name it as you like. 
+3. Make your changes in the branch
+4. Commit your changes and send a pull request
+
+And :tada:! You have contributed to this cool project!
```
Comment (athityakumar) [inline CONTRIBUTING.md]: -> And you've contributed to this cool project! 🎉

File: ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+## Do you wish a TV series?
```
Comment (athityakumar) [inline ISSUE_TEMPLATE.md]: -> Do you wish ***to add*** a TV Series?

**Comment by athityakumar** [comment]
Thanks a lot for working on this, @the-ethan-hunt. There are a few broad changes I'd like to add on -

1. Can we shift the "Working of the script" part to Contribution Guidelines?
2. Please refer to [this repo](https://github.com/athityakumar/colorls) for "better" Contribution Guidelines, Issue / PR templates and link them as required from the README.
3. Add yourself as a maintainer on the README, will you? 😄

**Comment by dhruvapte26** [comment]
@athityakumar , thanks for your insights and reviews on this. :smile: 
I have made the necessary changes.
>Add yourself as a maintainer on the README, will you? :smile: 

I am sorry,  but I did not understand this. Do I need to mention my name at the end of the `readme.md`? :sweat_smile:

**Comment by athityakumar** [comment]
@the-ethan-hunt - Yep, mention your name as a maintainer anywhere in the README. 😄

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
+Firstly, thanks for contributing to the project. :smile:
```
Comment (athityakumar) [inline CONTRIBUTING.md:3]: `# Contribution Guidelines` as the top-most section.

**Comment by athityakumar** [comment]
Just one minor change as suggested above, and we're good to merge. 😄

**Comment by dhruvapte26** [comment]
@athityakumar , I have added both of us in the maintainer section. I saw this on various good repositories that even the owner is mentioned as a maintainer. I have added our GitHub profiles as the links. Do tell me if you want your blog as the link. :smile:

**Pull request merged**